account_bank_statement_import_camt_oca/models/parser.py

```diff
@@ -254,3 +289,17 @@ def parse(self, data):
                     account_number = statement.pop('account_number')
                 statements.append(statement)
         return currency, account_number, statements
+
+    def _get_journal_use_ref_as_label_stmt_import(self):
+        res = False
+        merge_ref_name = self.env.context.get('merge_ref_name')
+
+        journal_id = self.env.context.get('journal_id')
+        if journal_id:
+            res = (
+                self.env['account.journal']
+                .browse(journal_id)
+                .use_ref_as_label_stmt_import
+            )
+        res = res or merge_ref_name
+        return res
```

Comment:
```suggestion
    def _get_journal_use_ref_as_label_stmt_import(self):
        # force_use_ref context is used only into unit test
        # because on unit test we don't have a defined journal
        force_use_ref = self.env.context.get('force_use_ref')
        if force_use_ref:
            return True
        journal_id = self.env.context.get('journal_id')
        if journal_id:
            journal = self.env['account.journal'].browse(journal_id)
            return journal.use_ref_as_label_stmt_import
        return False
```